.github/workflows/plan-release.yml

```diff
@@ -14,48 +15,47 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  check-plan:
-    name: "Check Release Plan"
+  is-this-a-release:
+    name: "Is this a release?"
     runs-on: ubuntu-latest
     outputs:
       command: ${{ steps.check-release.outputs.command }}
 
     steps:
-      - uses: actions/checkout@v5
+      - uses: actions/checkout@v4
```

Comment:
Is it necessary to downgrade `actions/checkout` and `actions/setup-node`?

Reply:
Not a question for here, but for the release-plan-setup -- none of these choices are "mine" but just "boilerplate so I don't have to maintain anything"

Reply:
real answer: v5 came out like.. this week, ya?